build_for_ultimaker.sh

```diff
@@ -56,19 +93,20 @@ while getopts ":cht" options; do
       ;;
     esac
 done
-shift "((OPTIND - 1))"
+shift "$((OPTIND - 1))"
 
 
 # Check build environment requirements
-if [ "${RUN_ENV_CHECK}" = "yes" ]; then
+if [ "${run_env_check}" = "yes" ]; then
     env_check
+    exit 0
 fi
 
 # Build the rootfs image
-./build.sh
+run_build
```

Comment:
i would put this below the run test.

run env and run test are run explicitly (without anything else) when requested. when doing a build, we always run the test and env check.